common/src/main/scala/com/aliyun/emr/rss/common/RssConf.scala

```diff
@@ -528,6 +528,10 @@ object RssConf extends Logging {
     conf.getTimeAsMs("rss.expire.nonEmptyDir.duration", "1d")
   }
 
+  def localWorkingDirName(conf: RssConf): String = {
```

Comment:
Better to rename to workingDirName, and HDFS/OSS should also respect this config @FMX

Reply:
> workingDirName

Yea， how about current